build.sbt

```
@@ -58,6 +58,9 @@ ThisBuild / githubWorkflowPublish := Seq(
   )
 )
 
+sonatypeProfileName := "com.lightbend.paradox"
```

I think this should be `com.lightbend` - or do we want to introduce `com.lightbend.paradox` to Sonatype?

```suggestion
sonatypeProfileName := "com.lightbend"
```